### PR TITLE
update: add input validation

### PR DESCRIPTION
**Pull request by PuneetP16**
Added: missing component input with validation.

### CODE REVIEW

File: src/components/input/input.html [hunk 1]
```diff
@@ -268,6 +268,135 @@ <h3>Input Component</h3>
...
+								placeholder="jonhdoe123"
+							/>
+						</div>
+					</form>
+				</script>
+			</div>
+
 			<h3>Complete Code Snippet for Different Input Components</h3>
 
 			<!-- snippet_container -->
```
Comment (deekshithmd) [inline src/components/input/input.html:402]: Code looks good, you can proceed to merge

**Review by deekshithmd — APPROVED** [review]

**Review by nikhiljugale007 — APPROVED** [review]
Did not find any issue in the code.
Good to merge.
Followed good naming convention.

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: src/components/input/input.css [hunk 2]
```diff
@@ -155,3 +155,23 @@ input:checked + .slider:before {
...
+	border: 2px solid var(--COLOR-DANGER);
+	outline: var(--COLOR-DANGER);
+}
+
+.input__success {
+	border: 2px solid var(--COLOR-SUCCESS);
+	outline: var(--COLOR-SUCCESS);
+}
+
+.input__success:hover {
```
Comment (pankaj-prog) [inline src/components/input/input.css:174]: class names are declarative. Good work

**Review by pankaj-prog — APPROVED** [review]

File: src/components/input/input.css [hunk 3]
```diff
@@ -155,3 +155,23 @@ input:checked + .slider:before {
...
+	outline: var(--COLOR-DANGER);
+}
+
+.input__error:hover {
+	border: 2px solid var(--COLOR-DANGER);
+	outline: var(--COLOR-DANGER);
+}
+
+.input__success {
+	border: 2px solid var(--COLOR-SUCCESS);
```
Comment (dev-ashishsethi) [inline src/components/input/input.css:170]: for border size you could have used a variable to make it consistent

File: src/components/input/input.css [hunk 4]
```diff
@@ -155,3 +155,23 @@ input:checked + .slider:before {
...
 }
+
+.input__error {
+	border: 2px solid var(--COLOR-DANGER);
+	outline: var(--COLOR-DANGER);
+}
+
+.input__error:hover {
+	border: 2px solid var(--COLOR-DANGER);
+	outline: var(--COLOR-DANGER);
```
Comment (dev-ashishsethi) [inline src/components/input/input.css:166]: you could have given a fallback value to variables using var(--variable name, another value)
Check this for refrence https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values

**Review by dev-ashishsethi — APPROVED** [review]
do check comment

**Pull request merged**